crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/CompareContributionService.java

```diff
@@ -22,14 +24,12 @@ public class CompareContributionService {
 
     private final MaatCourtDataService maatCourtDataService;
 
-    private final ContributionService contributionService;
-
     @Transactional
     public boolean shouldCreateContribution(CalculateContributionDTO calculateContributionDTO, ContributionResult contributionResult) {
         int repId = calculateContributionDTO.getRepId();
         RepOrderDTO repOrderDTO = calculateContributionDTO.getRepOrderDTO();
-        String magsCourtOutcome = calculateContributionDTO.getMagCourtOutcome() == null ? null
-                : calculateContributionDTO.getMagCourtOutcome().getOutcome();
+        MagCourtOutcome magsCourtOutcome = calculateContributionDTO.getMagCourtOutcome() == null ? null
```

Comment:
If `magsCourtOutcome == null` then we shouldn't need to specify the value of `null` as the assignment I don't think, we might be able to re-write this as:

`MagCourtOutcome magsCourtOutcome = calculateContributionDTO.getMagCourtOutcome();`

Reply:
Yeh realised as much as subsequently remove the line and just passing `calculateContributionDTO.getMagCourtOutcome()` into the `isMagsCourtOutcomeUnchanged` method.